### PR TITLE
CPNHUB-94 ADD: "Created externally" label for non user topics.

### PR DESCRIPTION
**Pull request by dev-tsoni**

### CODE REVIEW

File: assets/search/components/TopicForm.jsx [hunk 1]
```diff
@@ -39,14 +39,17 @@ const TopicForm = ({original, topic, save, onChange, globalTopicsEnabled, onSubs
                     readOnly={readOnly}
                 />
             )}
-            {!globalTopicsEnabled ? null : (
+            {!(globalTopicsEnabled && original.user) ? null : (
```
Comment (MarkLark86) [inline assets/search/components/TopicForm.jsx:42]: This will remove the `Share with my Company` checkbox when creating a new `Topic`.

**Review by MarkLark86 — CHANGES_REQUESTED** [review]

File: assets/search/components/TopicForm.jsx (hunk 1 above)
Comment (dev-tsoni) [reply assets/search/components/TopicForm.jsx:42]: When we create `topic` from UI, It is visible in `My Topics`. Then I have to click on `Share with my Company` option available to move it to `company topics`.
But when I create a topic from `API` without `User`, it get's a label `shared externally`.

Comment (MarkLark86) [reply assets/search/components/TopicForm.jsx:42]: The user should be able to create the Topic as a Company Topic when they initially create.
Therefor this checkbox should be available when creating a topic in the UI, not just updating the topic

Comment (dev-tsoni) [reply assets/search/components/TopicForm.jsx:42]: Ohh, okay. that means it is totally unnecessary.

Comment (MarkLark86) [reply assets/search/components/TopicForm.jsx]: No it is necessary to remove the checkbox, if the topic is an existing topic

**Review by MarkLark86 — APPROVED** [review]

**Pull request merged**